UnityProjects/MRTKDevTemplate/Assets/Scenes/Experimental/CanvasExampleSimpleActionButton.unity

```diff
@@ -3644,19 +3648,19 @@ PrefabInstance:
       objectReference: {fileID: 0}
     - target: {fileID: 209245566481228407, guid: cb2e4384d009baa4f804941a10b6781a, type: 3}
```

Comment:
@ms-RistoRK  please double check these changes.   Theis scene is changing more than just enabling the speech interactor.

Reply:
Double checked the scene, those extra changes in file were introduced by Unity Editor.  Just in case I repeated the process of copying .unity file from feature/XRI3 branch latest commit -> double-checked the new controllerless rig is used -> enabled MRTKSpeech rig component.
Resolving.